spec/controllers/sign_up/completions_controller_spec.rb

```diff
@@ -100,12 +100,15 @@
       @linker = instance_double(IdentityLinker)
       allow(@linker).to receive(:link_identity).and_return(true)
       allow(IdentityLinker).to receive(:new).and_return(@linker)
+      allow_any_instance_of(SignUp::CompletionsController).to \
```

Comment:
We should generally avoid using `allow_any_instance_of`. You should be able to use `allow(controller)` or `allow(subject)` here instead.

Reply:
yep was a quick hack after subject did not appear to work